src/Hl7.Fhir.Serialization.R4B.Tests/SerializationExcexptionHandlersJsonPoco.cs

```diff
@@ -0,0 +1,847 @@
+﻿using Hl7.Fhir.Model;
```

Comment:
When we have pulled this PR: https://github.com/FirelyTeam/firely-net-sdk/pull/2437, we can parameterize the tests with an `IFhirSerializationEngine`, which will halve the number of tests (since you can just pass in the engine, instead of having to create a POCO json + "old" json parser.